libs/agora/testing/src/lib/server/index.ts

```diff
@@ -0,0 +1 @@
+export * from './msw-server';
```

Comment:
added the /server folder to make it clearer on imports which 'server' is being exported